packages/examples/cvat/exchange-oracle/src/modules/oracle_webhook/helpers.py

```diff
@@ -0,0 +1,19 @@
+from typing import Dict
+
+from src.constants import Networks
+from src.modules.chain.web3 import sign_message, recover_signer
```

Comment:
```suggestion
from src.modules.chain.web3 import sign_message
```
This method is not used in this file